images.md

```diff
@@ -0,0 +1,22 @@
+# Lambda Instance images creation
+
+## Description
+
+Using the fokia library, an administrator is able to create updated images for the Lambda Instance Master and Slave nodes.
+
+## Usage
+
+1. To create the Lambda Instance images, one must use the fokia library. This can be done either on your own computer, after installing the required dependencies, or using an up-to-date Lambda Service VM.
```

Comment:
1. Yes, you can install fokia on any computer, as stated. It just must have the required dependencies.
2. The installation process is described in core/README.md.